src/common/assert.cpp

```
@@ -36,40 +36,46 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-
-
-const char* AssertFailedException::what() const throw() {
+const char *AssertFailedException::what() const throw()
+{
     return str.c_str();
 }
 
-AssertFailedException::AssertFailedException(std::string msg) :
-    str(msg) {}
-
+AssertFailedException::AssertFailedException(std::string msg) : str(msg) {}
```

Cppcheck (2.19.0)

[performance] Function parameter 'msg' should be passed by const reference.

(passedByValue)